### PR TITLE
Bar backend2

### PR DESCRIPTION
**Pull request by FabianAlleblas**
Ignore the migration files...

I added 2 properties to the entity plant, so the user can save the date she/he watered or fertilized the plant the last time.
With the initial request from the FE for the plantlist a function converts this dates into days and weeks left, so the FE can easy handle them.

### CODE REVIEW

File: backend/src/Repository/PlantRepository.php [hunk 1]
```diff
@@ -30,6 +30,28 @@ public function deletePlant(Plant $plant): void {
         $this->_em->flush();
     }
 
+    public function convertDate(Plant $plant): Plant {
```
Comment (leonp5) [inline backend/src/Repository/PlantRepository.php:33]: If i understand it right, you are not converting or?
Isn't it something like settingCarePeriod or so?

**Review by leonp5 — APPROVED** [review]

File: backend/src/Repository/PlantRepository.php (hunk 1 above)
Comment (FabianAlleblas) [reply backend/src/Repository/PlantRepository.php:33]: You're right, could called it setTimeLeft or something like that 👍

File: backend/src/Controller/PlantController.php [hunk 2]
```diff
@@ -37,6 +42,9 @@ public function createPlant(Request $request, PlantRepository $plantRepository,
             );
         }
 
+        $plant->setLastWatered(new \Datetime());
+        $plant->setLastFertilized(new \Datetime());
+
```
Comment (tine-krueger) [inline backend/src/Controller/PlantController.php:47]: Do you create a new plant on the same day you water and fertilize them?

File: backend/src/Entity/Plant.php [hunk 3]
```diff
@@ -79,5 +95,63 @@ public function setInfo(string $info): self
...
+    }
+    
+    public function getWeeksLeft(): ?int
+    {
+        return $this->weeksLeft;
+    }
+
+    public function setWeeksLeft(int $weeksLeft): self
+    {
+        $this->weeksLeft = $weeksLeft;
```
Comment (tine-krueger) [inline backend/src/Entity/Plant.php:153]: :-)

File: backend/src/Repository/PlantRepository.php [hunk 4]
```diff
@@ -30,6 +30,29 @@ public function deletePlant(Plant $plant): void {
         $this->_em->flush();
     }
 
+    public function setTimeLeft(Plant $plant): Plant {
```
Comment (web-devil) [inline backend/src/Repository/PlantRepository.php:33]: please create a service class for that

no database access => not into the repository

**Pull request merged**